scss/ScssLexer.g4

```diff
@@ -147,7 +147,7 @@ PrefixMatch    : '^=';
 SuffixMatch    : '$=';
 SubstringMatch : '*=';
 
-VendorPrefix: '-mox-' | '-webkit-' | '-o-';
+VendorPrefix: '-moz-' | '-webkit-' | '-o-';
```

Comment:
Please cover your changes with an example in [examples](https://github.com/antlr/grammars-v4/tree/master/scss/examples)